scipy-1.0/paper.tex

```diff
@@ -447,6 +447,14 @@ \section*{Project organisation and community}
 \textbf{Community beyond the SciPy library}
 
 \textbf{Maintainers and contributors}
+The SciPy project has approximately 15 active maintainers (people who review
+others' contributions, and in general do everything needed to ensure that
+the software and the project move forward).  It also has an additional ~100
+unique contributors for every 6-monthly release cycle.  Maintainers are
```

Comment:
Stressing the importance of maintainers after mentioning contributors might call to light that the paragraph does not mention the importance of contributors. Maybe a slight re-ordering would fix that?

The SciPy project has approximately 15 active maintainers: people who review
others' contributions, and in general do everything needed to ensure that
the software and the project move forward. Maintainers are
critical to the health of the project\cite{eghbal2016}; their skills and efforts largely determine how
fast the project moves forward, and they enable contributions
from a much larger group. Accordingly, the project also has an additional ~100
unique contributors for every 6-monthly release cycle. Anyone with the interest and skills
can become a contributor or maintainer; the SciPy Developer Guide provides guidance
on how to do that.

I think that helps a little without having to add a sentence about the importance of contributors.